scripts/gatsby-source-iiif/indexSearch.js

```diff
@@ -17,6 +17,8 @@ if (appConfig === 'local') {
   return
 }
 
+console.log(siteIndex, domain)
```

Comment:
If these are going to be left in for future debugging, let's change to:
```
console.log('siteIndex: ', siteIndex)
console.log('domain: ', domain)
```
Ditto with the `appConfig` log below?

Reply:
no i intend to remove them.